slides/history_digital_money.tex

```diff
@@ -0,0 +1,299 @@
+\column{0.7\textwidth}
+	\textbf{B-money - Wei Dai} \\
+	\vspace{0.5em}
+	\begin{small}
+	\begin{itemize}
+		\item Thought experiment
+		\item Assumption: Existence of an untraceable network.
+		\item Senders and receivers are identified only by digital pseudonyms. (i.e. public keys) 
+		\item Transaction legitimacy guaranteed by signature with associated private key.
+		\item Participants keep separate registers with the current balances of all pseudonyms.
+		\item Competitive money creation by solving numerical puzzles that are hard to compute but easy to verify.
+	\end{itemize}
+	\end{small}
+\end{columns}
+\end{frame}
+%%%	
+
+
+%%%
+\begin{frame}{Before Bitcoin}
+\begin{columns}
+\column{0.4\textwidth}
+\begin{figure}
+	\begin{tikzpicture}[scale=1]
+		\input{../assets/figures/before_bitcoin}
+		\filldraw[draw=black, fill = focus, thick] (0, -0.5 cm) circle (2pt);	
+	\end{tikzpicture}
+\end{figure}
+\column{0.7\textwidth}
+	\textbf{Reusable Proofs of Work - Hal Finney} \cite{finney2005}
+	\vspace{0.5em}
+	\begin{small}
+	\begin{itemize}
+		\item Combines ideas of Wei Dai and Adam Back.
+		\item A RPoW client can create a RPoW token by providing a proof-of-work string and signing with his private key.
+		\item Server can map that token to the signing key.
+		\item Client can give the token to another key by signing a transfer order to a public key.
+		\item Server registers the token as belonging to the corresponding private key.
+	\end{itemize}
+	\end{small}
+\end{columns}
+\end{frame}
+%%%	
+
+
+%%%
+\begin{frame}{Before Bitcoin}
+\begin{columns}
+\column{0.4\textwidth}
+\begin{figure}
+	\begin{tikzpicture}[scale=1]
+		\input{../assets/figures/before_bitcoin}
+		\filldraw[draw=black, fill = focus, thick] (0, -0.5 cm) circle (2pt);
+	\end{tikzpicture}
+\end{figure}
+\column{0.7\textwidth}
+	\textbf{Bit Gold - Nick Szabo}
+	\vspace{0.5em}
+	\begin{small}
+	\begin{itemize}
+		\item Describes the combination of the proof-of-work algorithm for competitive money creation.
+		\item Computing power is used to collateralise a public ledger, which grows into a chain of blocks through a reflexive reference.
+		\item "Thus, it would be very nice if there were a protocol whereby unforgeably costly bits could be created online with minimal dependence on trusted third parties, and then securely stored, transferred, and assayed with similar minimal trust. Bit gold." \cite{szabo2005}
+	\end{itemize}
+	\end{small}
+\end{columns}
+\end{frame}
+%%%	
+
+
+%%%
+\begin{frame}{After Bitcoin: Forks and Altcoins}
+	\begin{columns}
+		\column{0.4\linewidth}
+		\begin{figure}
+			\begin{tikzpicture}[scale=1]
+				\input{../assets/figures/after_bitcoin}
+				\filldraw[draw=black, fill = focus, thick] (0, 3.5 cm) circle (2pt);
+				\filldraw[draw=black, fill = focus, thick] (0, 1.5 cm) circle (2pt);
+			\end{tikzpicture}
+		\end{figure}
+		\column{0.6\linewidth}
+			\begin{tikzpicture}
+				\node (Dogecoin) {\includegraphics[scale=0.13]{../assets/images/doge}};
```

Comment:
Use cryptocurrency symbols in accordance with style of the slides, i.e. black and white logos. See symbols used in #44